packages/pipelines/package.json

```diff
@@ -42,26 +42,31 @@
     "@aonic-ui/jest-config": "*",
     "@aonic-ui/rollup-config": "*",
     "@aonic-ui/typescript-config": "*",
-    "@testing-library/react": "^14.1.2",
-    "@testing-library/jest-dom": "^6.2.0",
+    "@openshift-console/dynamic-plugin-sdk": "1.2.0",
+    "@openshift-console/dynamic-plugin-sdk-webpack": "1.1.0",
```

Comment:
This looks very specific to console, the above packages internally reference utils, methods from `@console/internal`, which is only available in console apps and non-console apps (Konflux) will get an error as it will not be able to resolve `@console/internal`. I think we should make this work regardless of the app it is used in. @rohitkrai03 thoughts?

Reply:
As discussed in call, remove this dependancies and accept the K8sget/watch hooks as parameters. This way these hooks will be generic and consumers can decide which hook to pass.

Reply:
Updated the PR. PTAL.